junifer/testing/tests/test_oasisvmbtesting_datagrabber.py

```diff
@@ -3,10 +3,10 @@
 # Authors: Federico Raimondo <f.raimondo@fz-juelich.de>
 # License: AGPL
 
-from junifer.testing.datagrabbers import OasisVBMTestingDatagrabber
+from junifer.testing.datagrabbers import OasisVBMTestingDataGrabber
 
 
-def test_OasisVBMTestingDatagrabber() -> None:
+def test_OasisVBMTestingDataGrabber() -> None:
     """Test Oasis VBM Testing datagrabber."""
```

Comment:
Should it be `DataGrabber` in the docstring here?

Reply:
I see your point, I'll update the docstrings.